modules/howtos/pages/analytics-using-sdk.adoc

```diff
@@ -133,3 +133,27 @@ include::example$analytics.rb[tag=scanconsistency]
 ----
 
 
+
+== Scoped Queries on Named Collections
+
+Given a dataset created against a collection, for example:
+
+[source,n1ql]
+----
+CREATE DATASET `airports-collection` ON `travel-sample`.inventory.airport;
+----
+
+You can run a query as follows:
+
+[source,ruby]
+----
+include::../examples/analytics.rb[tag=handle-collection]
```

Comment:
I'm not sure what extra macro power Antora derives from using example$ in place of ../example/
(I'd need to rtfm ;) )
but I'm pretty sure that there was a reason beyond cargo-culting why we do this.